src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -147,6 +161,23 @@ static MsQuicApi()
         private static bool IsWindowsVersionSupported() => OperatingSystem.IsWindowsVersionAtLeast(MinWindowsVersion.Major,
             MinWindowsVersion.Minor, MinWindowsVersion.Build, MinWindowsVersion.Revision);
 
+        private static bool IsTls1_3Disabled()
+        {
+#if WINDOWS
```

Comment:
I'm not sure if we need this. Linker may be smart enough to trim it away. If we do, we seems to use `TARGET_WINDOWS` in other places like this. That makes it work for cross-targeting e.g. building Windows bit on Linux and vice versa. (needs small project file change)

Reply:
I gave it a try and it didn't look like the linker could trim the Windows only bits, so I've added TARGET_WINDOWS